api/PclusterApiHandler.py

```diff
@@ -680,44 +684,10 @@ def check_tags(self, inactive_tags):
         return {"message": str(e)}, 500
 
 # Activate parallelcluster tags in cost explorer
-def activate_tags(self, cost_explorer):
+def activate_tags():
     try:
-        updated_tags = cost_explorer.update_cost_allocation_tags_status(
-            CostAllocationTagsStatus=[
-                {
-                    'TagKey': 'parallelcluster:attributes',
-                    'Status': 'Active'
-                },
-                {
-                    'TagKey': 'parallelcluster:cluster-name',
-                    'Status': 'Active'
-                },
-                {
-                    'TagKey': 'parallelcluster:compute-resource-name',
-                    'Status': 'Active'
-                },
-                {
-                    'TagKey': 'parallelcluster:filesystem',
-                    'Status': 'Active'
-                },
-                {
-                    'TagKey': 'parallelcluster:node-type',
-                    'Status': 'Active'
-                },
-                {
-                    'TagKey': 'parallelcluster:queue-name',
-                    'Status': 'Active'
-                },
-                {
-                    'TagKey': 'parallelcluster:resource',
-                    'Status': 'Active'
-                },
-                {
-                    'TagKey': 'parallelcluster:version',
-                    'Status': 'Active'
-                }
-            ]
-        )
+        for tag in PCluster_tags:
+            updated_tags = cost_explorer.CostAllocationTagStatus.append({'Tagkey': tag, 'Value': 'Active'})
```

Comment:
We still need to call the API. I was just referring to the way we pass in the list of tags. 

Let's use a python list comprehension here: https://www.w3schools.com/python/python_lists_comprehension.asp